fhir-persistence-schema/src/test/java/com/ibm/fhir/schema/control/FhirSchemaServiceTest.java

```diff
@@ -140,8 +116,11 @@ public void testDrop() {
         // Print the statements instead of executing them against a database
         PrintTarget tgt = new PrintTarget(null, logger.isLoggable(Level.FINE));
 
-        // Pretend that our target is a DB2 database
-        Db2Adapter adapter = new Db2Adapter(tgt);
+        // Pretend that our target is a Derby database
+        DerbyAdapter adapter = new DerbyAdapter(tgt) {
```

Comment:
in other cases we replaced with Postgres...any reason to target derby from this one?

Reply:
Yeah, because Postgres requires a whole bunch of `doesXXExist` methods to be overridden which are currently private.